src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
If that is a Resharper option, feel free to add it as a part of this PR or as a separate PR if you prefer. I stopped using Resharper so stopped keeping the dotsettings up to date.